CONTRIBUTING.md

```diff
@@ -1,107 +1,111 @@
+running, please look at [LOCAL.md](/LOCAL.MD) for setup instructions.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[CONTRIBUTING.md Code of Conduct](blob/master/CODE_OF_CONDUCT.md). By
+participating you are expected to uphold this code. Please report unacceptable
+behavior to <placeholder@typehero.dev>
+
+## I Want to Contribute
+
+> ### Legal Notice
+>
+> When contributing to this project, you must agree that you have authored 100%
+> of the content, that you have the necessary rights to the content and that the
+> content you contribute may be provided under the project's license.
+
+### Reporting Bugs
+
+#### Before Submitting a Bug Report
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Therefore, we ask you to investigate carefully, collect
+information, and describe the issue in detail _in your report_. Please complete
+the following steps in advance to help us fix any potential bug as fast as
+possible:
+
+1. Document the page(s) the bug is happening on.
+2. Take applicable Screenshots if necessary, and create any relevant stack
+   traces (if possible)
+3. If you are experiencing this bug on your local, ensure that it is not do to
+   an incorrect local configuration.
+4. Check if a similar issue already exists in the
+   [bug tracker](issues?q=label%3A:bug)
+
+#### How do I Submit a Good Bug Report?
+
+> ### You must never report security related issues, vulnerabilities, or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead, sensitive bugs must be sent by email to [bugs.sensitive@typehero.dev](mailto:bugs.sensitive@typehero.dev)
+
+We use GitHub issues to track bugs and errors. If you run into an issue with the
+project:
+
+- Open an [Issue](/issues/new).
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the _reproduction
+  steps_ that someone else can follow to recreate the issue on their own. This
+  usually includes your code. For good bug reports you should isolate the
+  problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it is filed:
+
+- The project team will label that issue accordingly.
+- A team member will try to reproduce the issue with your provided steps. If
+  there are no reproduction steps or no obvious way to reproduce the issue, the
+  team will ask you for those steps and mark the issue as `needs-repro`. Bugs
+  with the `needs-repro` tag will not be addressed until they are reproduced.
+- If the team is able to reproduce the issue, it will be marked as `needs-fix`,
+  as well as possibly other tags (such as `critical`), and the issue will be
+  left to be [implemented by someone](#your-first-code-contribution)
+
+### Suggestiong Enhancements
+
+This section guides you through submitting an enhancement suggestion for
```

Comment:
id mention starting conversations in discord. we want to funnel traffic to discord as much as possible

Reply:
Do you mean that they should start the enhancement discussions on discord instead of github?

Reply:
discord is an alternative. people will have many ideas. we can use discord forums